rules/k8s_audit_rules.yaml

```diff
@@ -40,13 +40,16 @@
 
 # If you wish to restrict activity to a specific set of users, override/append to this list.
 # users created by kops are included
+- list: kube_apiserver_healthcheck_users
+  items: ["kube-apiserver-healthcheck"]
+
 - list: vertical_pod_autoscaler_users
   items: ["vpa-recommender", "vpa-updater"]
 
 - list: allowed_k8s_users
   items: [
     "minikube", "minikube-user", "kubelet", "kops", "admin", "kube", "kube-proxy",
-    vertical_pod_autoscaler_users,
+    kube_apiserver_healthcheck_users, vertical_pod_autoscaler_users,
```

Comment:
can we add `kube-apiserver-healthcheck` directly to the list `allowed_k8s_users`?

Reply:
that would work

Reply:
@Kaizhe still marked as "changes requested", thanks :)